submissions/pozzitive11/popup/index.html

```diff
@@ -0,0 +1,166 @@
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>Hooli-style Popup</title>
+  <link rel="icon" type="image/x-icon" href="/images/favicon.png">
+  <link rel="stylesheet" href="css/style.css">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;700&display=swap" rel="stylesheet">
+</head>
+
+<body>
+  <header class="header">
+    <div class="container">
+      <nav class="nav" title="Main navigation">
+        <ul class="list-reset nav__list">
+          <li class="nav__item">
+            <a href="#" class="nav__link">
+              Mail
+            </a>
+          </li>
+          <li class="nav__item">
+            <a href="#" class="nav__link">
+              Images
+            </a>
+          </li>
+          <li class="nav__item nav__item-btn">
+            <label for="menu__input" class="menu__input" tabindex="0">
+              <img src="images/popup-button.png" alt="popup-button" class="menu__input-icon">
+            </label>
+            <input type="checkbox" id="menu__input" class="popup__btn" />
+            <div class="popup">
+              <div class="popup__inner">
+                <div class="main-content">
+                  <ul class="list-reset popup__list">
+                    <li class="popup__item">
+                      <a class="popup__link" href="#">
+                        <img src="images/calendar.png" alt="Calendar">
+                        <p class="popup__item-text">Calendar</p>
+                      </a>
+                    </li>
+                    <li class="popup__item">
+                      <a class="popup__link" href="#">
+                        <img src="images/mail.png" alt="Mail">
+                        <p class="popup__item-text">Mail</p>
+                      </a>
+                    </li>
+                    <li class="popup__item">
+                      <a class="popup__link" href="#">
+                        <img src="images/news.png" alt="News">
+                        <p class="popup__item-text">News</p>
+                      </a>
+                    </li>
+                    <li class="popup__item">
+                      <a class="popup__link" href="#">
+                        <img src="images/maps.png" alt="Map">
+                        <p class="popup__item-text">Map</p>
+                      </a>
+                    </li>
+                    <li class="popup__item">
+                      <a class="popup__link" href="#">
+                        <img src="images/files.png" alt="Files">
+                        <p class="popup__item-text">Files</p>
+                      </a>
+                    </li>
+                    <li class="popup__item">
+                      <a class="popup__link" href="#">
+                        <img src="images/contacts.png" alt="Contacts">
```

Comment:
Why do you leave the filled `alt` attribute for all these icons, if you remove it in previous commits?

Reply:
when the `img` tag is used, isn't an `alt `required?

Reply:
Sorry for russians but it's a good [article](https://htmlacademy.ru/blog/articles/alt-text)

Reply:
that is, these images are considered _"icons in a button"_ and should not write something in the `alt` tag for them